views/flower.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+
+        <record id="flower_form_view" model="ir.ui.view">
+            <field name="name">flower.form</field>
+            <field name="model">flower</field>
+            <field name="arch" type="xml">
+                <form string="Flower">
+                    <sheet>
+                        <div class="oe_title">
+                            <label for="name"/>
+                            <h1>
+                                <field name="name" placeholder="Name"/>
+                            </h1>
+                        </div>
+                        <group>
+                            <group>
+                                <field name="scientific_name"/>
+                                <field name="season_date_start"/>
+                                <field name="season_date_end"/>
+                            </group>
+                            <group>
+                                <field name="watering_frequency"/>
+                                <field name="watering_amount"/>
+                            </group>
+                        </group>
+                    </sheet>
+                </form>
+            </field>
+        </record>
+
+        <record id="flower_tree_view" model="ir.ui.view">
+            <field name="name">flower.tree</field>
+            <field name="model">flower</field>
+            <field name="arch" type="xml">
+                <tree string="Flower">
+                    <field name="name"/>
+                    <field name="scientific_name"/>
+                    <field name="season_date_start"/>
+                    <field name="season_date_end"/>
+                    <field name="watering_frequency"/>
+                    <field name="watering_amount"/>
+                </tree>
+            </field>
+        </record>
+
+        <record id="flower_search_view" model="ir.ui.view">
+            <field name="name">flower.search</field>
+            <field name="model">flower</field>
+            <field name="arch" type="xml">
+                <search string="Flower">
+                    <field name="name"/>
+                    <field name="scientific_name"/>
+                    <field name="season_date_start"/>
+                    <field name="season_date_end"/>
+                    <field name="watering_frequency"/>
+                    <field name="watering_amount"/>
+                    <group expand="1" string="Group By">
+                        <filter string="Watering Frequency" name="watering_frequency" domain="[]"
```

Comment:
I'm guessing the domain is an oversight? Since it doesn't really make sense to have a domain on an Integer field.